e2e-tests/development-runtime/cypress/integration/hot-reloading/error-handling/page-query-result-runtime-error.js

```diff
@@ -0,0 +1,85 @@
+Cypress.on(
+  `uncaught:exception`,
+  (err, runnable) =>
+    // returning false here prevents Cypress from
+    // failing the test
+    false
+)
+
+before(() => {
+  cy.exec(
+    `npm run update -- --file content/error-recovery/page-query.json --restore`
+  )
+  cy.exec(
+    `npm run update -- --file src/pages/error-handling/page-query-result-runtime-error.js --restore`
+  )
+})
+
+after(() => {
+  cy.exec(
+    `npm run update -- --file content/error-recovery/page-query.json --restore`
+  )
+  cy.exec(
+    `npm run update -- --file src/pages/error-handling/page-query-result-runtime-error.js --restore`
+  )
+})
+
+const errorPlaceholder = `false`
+const errorReplacement = `true`
+
+describe(`testing error overlay and ability to automatically recover runtime errors cause by content changes (page queries variant)`, () => {
+  it(`displays content initially (no errors yet)`, () => {
+    cy.visit(`/error-handling/page-query-result-runtime-error/`, {
+      // Hacky way to disable "uncaught:exception" message in error message itself
+      // See https://github.com/cypress-io/cypress/issues/254#issuecomment-292190924
+      onBeforeLoad: win => {
```

Comment:
Without this change the runtime error would be a long string of "blabla Cypress caught an exception blabla then original error blabla" - so the string I want to assert has the original error but a lot of other text around it. With this the text inside `accordion__item__title` only has the error text, not more